src/cocaine/burlak/burlak.py

```diff
@@ -38,13 +38,11 @@
 from .mixins import *
 
 
-CONTROL_RETRY_ATTEMPTS = 3
-
 DEFAULT_RETRY_TIMEOUT_SEC = 15
 DEFAULT_UNKNOWN_VERSIONS = 1
 
 DEFAULT_RETRY_ATTEMPTS = 4
-DEFAULT_RETRY_EXP_BASE_SEC = 4
+DEFAULT_RETRY_EXP_BASE_SEC = 30
```

Comment:
Lets make this constant considerable small  (from 5 up to 10 seconds would be enought) or it will lead to long Control coroutine lock in case of control failure.